Cargo.toml

```diff
@@ -1,19 +1,19 @@
 [workspace]
 members = [
-  "ic-exports",
-  "ic-factory",
-  "ic-auction",
-  "ic-metrics",
-  "ic-helpers",
-  "ic-stable-structures",
-  "ic-storage",
-  "ic-storage/ic-storage-derive",
-  "ic-canister/ic-canister",
-  "ic-canister/ic-canister-macros",
-  "ic-canister/tests/canister-a",
-  "ic-canister/tests/canister-b",
-  "ic-canister/tests/canister-c",
-  "ic-canister/tests/canister-d",
+    "ic-exports",
+    "ic-factory",
+    "ic-auction",
+    "ic-metrics",
+    "ic-helpers",
+    "ic-stable-storage",
```

Comment:
There is no `ic-stable-storage` package in the workspace now. But the `ic-stable-structures` is present.

Reply:
Already fixed)